adapters/silvermob/silvermob.go

```diff
@@ -121,8 +133,14 @@ func (a *SilverMobAdapter) getImpressionExt(imp *openrtb2.Imp) (*openrtb_ext.Ext
 }
 
 func (a *SilverMobAdapter) buildEndpointURL(params *openrtb_ext.ExtSilverMob) (string, error) {
-	endpointParams := macros.EndpointTemplateParams{ZoneID: params.ZoneID, Host: params.Host}
-	return macros.ResolveMacros(a.endpoint, endpointParams)
+	if isValidHost(params.Host) {
```

Comment:
Please be mindful that your adapter has been flagged as it is currently using dynamic hosts. Please refer to the github issue here - https://github.com/prebid/prebid-server/issues/2612

Reply:
Yes, that's why we added this validation. Is it enough?

Reply:
You should strive to render the host non-dynamic by excluding the host parameter from the domain and use it as a query parameter instead. The domain name should remain constant.

Reply:
Ok, we will select a vendor for geo DNS and publish update within a couple of weeks

Reply:
@freemmy Please advise if you would like to make changes for geo DNS in this PR or a separate PR.

Reply:
I thing the next PR since it takes some time